presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/optimization/function/JdbcTranslationUtil.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.jdbc.optimization.function;
+
+import com.facebook.presto.plugin.jdbc.optimization.JdbcSql;
+import com.facebook.presto.spi.relation.ConstantExpression;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+
+public class JdbcTranslationUtil
+{
+    private JdbcTranslationUtil()
+    {
+    }
+
+    public static String infixOperation(String operator, JdbcSql left, JdbcSql right)
+    {
+        return String.format("%s %s %s", left.getSql(), operator, right.getSql());
```

Comment:
You need the (): (%s %s %s) to make sure the operator precedence is correct. Could you add tests in `TestJdbcIntegrationSmokeTest` to make sure the pushdown yields the right result?